pyspecProcScripts/correlation_alignment.py

```diff
@@ -17,7 +17,9 @@ def to_percent(y, position):
 def correl_align(
     s_orig,
     tol=1e-4,
-    indirect_dim="indirect",
+    indirect_dim=None,
+    repeat_dims=["indirect"],
+    non_repeat_dims=[""],
```

Comment:
unless specified this is empty but length is 1

Reply:
no -- default to an empty list